### PR TITLE
feat(@vtmn/web-components): add link component

### PR DESCRIPTION
**Pull request by GaspardMathon**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x] Make sure you are making a pull request against the **main branch** (left side).
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Description

- Add link web component with 3 sizes, standalone variant

## Does this introduce a breaking change?

- No

## Other information

❤️ Thank you!

### CODE REVIEW

**Comment by thollander** [comment]
I suggest to pass the #290 as it introduces a sharing of the Component Story Format (CSF) between all framework implementations. 
This PR could be rebased after with this change.

File: packages/showcases/web-components/stories/components/vtmn-link/vtmn-link.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import { html } from 'lit-html';
+import { spread } from '@open-wc/lit-helpers';
+
+export default {
+  title: 'Components/Link',
```
Comment (lauthieb) [inline packages/showcases/web-components/stories/components/vtmn-link/vtmn-link.stories.tsx:5]:
```suggestion
  title: 'Components/vtmn-link',
```
And you can also change it for Button if you want :)

File: packages/sources/web-components/src/components/vtmn-link/vtmn-link.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+   * The size of the link.
+   * @default 'medium'
+   */
+  @Prop() size: 'small' | 'medium' | 'large' = 'medium';
+
+  /**
+   * Is the link standalone or not
+   * @default false
+   */
+  @Prop({ attribute: 'standalone' }) isStandalone: boolean = false;
```
Comment (lauthieb) [inline packages/sources/web-components/src/components/vtmn-link/vtmn-link.tsx:19]:
```suggestion
  @Prop() isStandalone: boolean = false;
```
Like we did with React, it's okay.

File: packages/showcases/web-components/stories/components/vtmn-link/vtmn-link.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    },
+    target: {
+      type: { name: 'string', required: false },
+      description: 'The target of the link.',
+      defaultValue: '_blank',
+      control: {
+        type: 'text',
+      },
+    },
+    standalone: {
```
Comment (lauthieb) [inline packages/showcases/web-components/stories/components/vtmn-link/vtmn-link.stories.tsx:24]:
```suggestion
    isStandalone: {
```

**Review by lauthieb — CHANGES_REQUESTED** [review]

**Comment by lauthieb** [comment]
Thanks @GaspardMathon 
Agree w/ @thollander

**Comment by lauthieb** [comment]
@GaspardMathon @thollander @jleveugle @gabbloquet it could be nice for Web Components to extend HTMLElement attributes instead of declaring `href` and `target` props... But I didn't find how it could be possible to spread other props in render method of our Web Components implemented with Stencil :/

File: packages/sources/web-components/src/components/vtmn-link/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# vtmn-button
```
Comment (lauthieb) [inline packages/sources/web-components/src/components/vtmn-link/README.md:1]:
```suggestion
# vtmn-link
```

**Review by lauthieb — CHANGES_REQUESTED** [review]

File: packages/showcases/web-components/stories/components/vtmn-link/vtmn-link.stories.tsx (hunk 1 above)
Comment (GaspardMathon) [reply packages/showcases/web-components/stories/components/vtmn-link/vtmn-link.stories.tsx]: As it is now shared in the core folder, how to name it ? VtmnLink or vtmn-link ?

Currently, it is labelled as VtmnLink for information.

File: packages/showcases/web-components/stories/components/vtmn-link/vtmn-link.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+      description: 'The size of the link.',
+      defaultValue: 'medium',
+      control: {
+        type: 'select',
+        options: ['small', 'medium', 'large'],
+      },
+    },
+  },
+  parameters: {
+    backgrounds: { default: 'white' },
```
Comment (gabbloquet) [inline packages/showcases/web-components/stories/components/vtmn-link/vtmn-link.stories.tsx:43]: I think we use grey ? or maybe that's only for React ?

File: packages/sources/web-components/src/components/vtmn-link/vtmn-link.tsx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+   * The hypertext link
+   * @default '#'
+   */
+  @Prop() href: string = '#';
+
+  /**
+   * The target of the link
+   * @default '_blank'
+   */
+  @Prop() target: string = '_target';
```
Comment (gabbloquet) [inline packages/sources/web-components/src/components/vtmn-link/vtmn-link.tsx:31]:
```suggestion
  @Prop() target: string = '_blank';
```

File: packages/sources/web-components/src/components/vtmn-link/vtmn-link.tsx [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+        href={this.href}
+        target={this.target}
+        class={[
+          'vtmn-link',
+          `vtmn-link_size--${this.size}`,
+          this.isStandalone && 'vtmn-link--standalone',
+        ]
+          .filter(Boolean)
+          .join(' ')}>
+        <slot></slot>
```
Comment (gabbloquet) [inline packages/sources/web-components/src/components/vtmn-link/vtmn-link.tsx:45]: What's the purpose of slot ?

**Review by gabbloquet — COMMENTED** [review]
Only a request change on _blank, else question to learn new things, thanks for this contrib' one time again ;)

File: packages/sources/web-components/src/components/vtmn-link/vtmn-link.tsx (hunk 7 above)
Comment (lauthieb) [reply packages/sources/web-components/src/components/vtmn-link/vtmn-link.tsx:45]: It's like props.children in React :)

File: packages/showcases/core/src/components/VtmnLink.csf.js [hunk 8]
```diff
@@ -11,8 +11,10 @@ export default (component) => ({
     target: {
       type: { name: 'string', required: false },
       description: 'The target of the link.',
-      defaultValue: '_blank',
-      control: { type: 'text' },
+      defaultValue: '_self',
```
Comment (lauthieb) [inline packages/showcases/core/src/components/VtmnLink.csf.js:14]:
```suggestion
```

File: packages/sources/web-components/src/components/vtmn-link/vtmn-link.tsx [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+   * The size of the link.
+   * @default 'medium'
+   */
+  @Prop() size: 'small' | 'medium' | 'large' = 'medium';
+
+  /**
+   * Is the link standalone or not
+   * @default false
+   */
+  @Prop({ attribute: 'isstandalone' }) isStandalone: boolean = false;
```
Comment (lauthieb) [inline packages/sources/web-components/src/components/vtmn-link/vtmn-link.tsx:19]:
```suggestion
  @Prop() isStandalone: boolean = false;
```

File: packages/sources/web-components/src/components/vtmn-link/vtmn-link.tsx [hunk 10]
```diff
@@ -0,0 +1,49 @@
+import { Component, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'vtmn-link',
+  styleUrl: 'vtmn-link.pcss',
+  shadow: true,
```
Comment (lauthieb) [inline packages/sources/web-components/src/components/vtmn-link/vtmn-link.tsx:6]:
```suggestion
```

**Review by lauthieb — CHANGES_REQUESTED** [review]

**Review by lauthieb — APPROVED** [review]

**Pull request merged**